bench_test.go

```diff
@@ -79,25 +94,24 @@ func BenchmarkSpan_100BaggageItems(b *testing.B) {
 	benchmarkWithOps(b, 0, 0, 100)
 }
 
-func BenchmarkTrimmedSpan_100Events_100Tags_100BaggageItems(b *testing.B) {
-	var r CountingRecorder
-	t, err := NewTracer(
-		&r,
-		TrimUnsampledSpans(true),
-		WithSampler(neverSample),
-		TraceID128Bit(true),
-	)
-	if err != nil {
-		b.Fatalf("Unable to create Tracer: %+v", err)
-	}
-	benchmarkWithOpsAndCB(b, func() opentracing.Span {
-		sp := t.StartSpan("test")
-		return sp
-	}, 100, 100, 100)
-	if int(r) != b.N {
-		b.Fatalf("missing traces: expected %d, got %d", b.N, r)
-	}
-}
+// func BenchmarkTrimmedSpan_100Events_100Tags_100BaggageItems(b *testing.B) {
+// 	var r CountingRecorder
+// 	t, err := NewTracer(
+// 		&r,
+// 		zipkin.WithSampler(zipkin.NeverSample),
```

Comment:
As far as I see on current zipkin-go-ot, even when noop the span gets into the recorder, is that true? https://github.com/openzipkin-contrib/zipkin-go-opentracing/blob/master/span_test.go#L167